Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM node:latest
-WORKDIR /app
+WORKDIR /src
 COPY package*.json ./
 RUN npm install
 COPY . .
```

Comment:
you need to update the cmd line to something like :
CMD ["node", "dist/app.js"]